analyzer-core/src/main/java/ffb/analyzer/core/queue/PriorityTaskHandler.java

```diff
@@ -0,0 +1,15 @@
+package ffb.analyzer.core.queue;
+
+abstract class PriorityTaskHandler {
+    private PriorityTaskHandler handler;
+
+    public PriorityTaskHandler(PriorityTaskHandler handler) {
```

Comment:
What's the reason we're passing another `PriorityTaskHandler` here?

Reply:
Was meant to be recursive, but this isn't used anymore so I removed PriorityTaskHandler altogether.